com.outerwildsmods.owmods_gui.yml

```diff
@@ -0,0 +1,41 @@
+app-id: com.outerwildsmods.owmods_gui
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.mono6
+command: outer-wilds-mod-manager
+rename-icon: outer-wilds-mod-manager
+finish-args:
+    - --socket=fallback-x11
+    - --socket=wayland
```

Comment:
does it really support wayland?

Reply:
I'm basing this manifest off of [another app that uses tauri](https://github.com/flathub/in.cinny.Cinny/blob/master/in.cinny.Cinny.yml), which has that marked. And I've tested it on Wayland itself under plasma.

Reply:
I decided to switch back to Wayland to test this and yes it does work without issue.